spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/ldap/embedded/EmbeddedLdapAutoConfiguration.java

```diff
@@ -114,14 +114,13 @@ public InMemoryDirectoryServer directoryServer() throws LDAPException {
 			config.setSchema(null);
 		}
 		else if (this.embeddedProperties.getValidation().getSchema() != null) {
-			Resource schemaLocation = this.embeddedProperties.getValidation().getSchema();
 			try {
 				config.setSchema(Schema.mergeSchemas(Schema.getDefaultStandardSchema(),
-						Schema.getSchema(schemaLocation.getFile())));
+						Schema.getSchema(
```

Comment:
That's not going to work. That code expects the String to point to a File. I've opened an issue to get an `InputStream` variant so I'd rather wait for that.